scripts/generator/graphql_generator/csharp/SDK/iOS/SummaryItem.cs.erb

```diff
@@ -0,0 +1,19 @@
+#if UNITY_IPHONE
+namespace <%= namespace %>.SDK.iOS {
+    using System.Collections;
+    using System;
+
+    [Serializable]
+    public class SummaryItem: Serializable {
+        #pragma warning disable 0414
+        public string Label;
```

Comment:
Since the class is `Serializable`, if we leave them public without readonly, we can utilize `JsonUtility.FromJson<SummaryItem>`. Although we don't use that function right now, it would be confusing if we remove that functionality from a `Serializable`